azurerm/resource_arm_container_group_test.go

```diff
@@ -469,6 +475,12 @@ resource "azurerm_container_group" "test" {
 		"foo"  = "bar"
 		"foo1" = "bar1"
 	}
+
+	secure_environment_variables {
+		"secureFoo"  = "secureBar"
+		"secureFoo1" = "secureBar1"
+	}
```

Comment:
Can you replace tabs with whitespaces ? I know there're many inconsistency within this file, but at least we should be consistency for this particular configuration.